whichone/static/js/data.js

```diff
@@ -83,3 +83,15 @@ async function getSpotifyData() {
   await request("top_artists", compareArtists);
   await request("top_tracks", getAudioFeatures);
 }
+
+
+function deleteAllCookies() {
+  let cookies = document.cookie.split(";");
+
+  for (let i = 0; i < cookies.length; i++) {
+    let cookie = cookies[i];
```

Comment:
Might want to look into https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

Tends to be good practice to use const when variable does not get changed.

Also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach could simplify the loop a little.

Both optional changes that would be worth looking into a little

Reply:
@connor-settle I've updated this now in the latest commit